README.md

```diff
@@ -14,95 +17,82 @@ or set of validation rules (best practices).
 [![][error prone img]][error prone]
 [![][pitest img]][pitest]
 [![][checker framework img]][checker framework]
-
 [![][dependabot img]][dependabot]
-
 [![][mavenbadge img]][mavenbadge]
 [![][sonar img]][sonar]
-
 [![][release notes/version img]][release notes/version]
-
 [![][closed issues img]][closed issues]
 [![][link check img]][link check]
-
 [![][milestone img]][milestone]
 
-Contributors chat: [![][matrix_con img]][matrix_con]
-
-The latest release version can be found at
-[GitHub releases](https://github.com/checkstyle/checkstyle/releases/)
-or at [Maven repo](https://repo1.maven.org/maven2/com/puppycrawl/tools/checkstyle/).
-
-Each-commit builds of maven artifacts can be found at
-[Maven Snapshot repository](https://oss.sonatype.org/content/repositories/snapshots/com/puppycrawl/tools/checkstyle/).
-
-Documentation is available in HTML format, see https://checkstyle.org/checks.html.
-
-## Build instructions and Contribution
-
-[Build instructions](https://checkstyle.org/contributing.html#Build)
+## Table of Contents
+1. [Quick Start](#quick-start)
+2. [Contributing](#contributing)
+3. [Feedback and Support](#feedback-and-support)
+4. [Project Reports](#project-reports)
+5. [Sponsor Checkstyle](#sponsor-checkstyle)
+6. [Licensing](#licensing)
 
-[Setup IDE for development](https://checkstyle.org/beginning_development.html)
+## Quick Start
```

Comment:
This is probably a good first pass, but I feel quick start is way too quick. Usually I like to see small and quick examples and such on the README on how to use the repo. I usually go to documentation afterwards once I am looking for more complex or specific items.

Reply:
We can make CLI output for some single violation over short code.

Reply:
small example is placed.